db/migrate/20200508155813_populate_response_cached_json.rb

```diff
@@ -40,8 +40,8 @@ def cache_responses(responses)
     total = responses.count
     start = Time.zone.now
     num_procs = ENV["NUM_PROCS"] ? ENV["NUM_PROCS"].to_i : Etc.nprocessors
-    Parallel.each_with_index(responses.find_each, in_processes: num_procs,
-                                                  isolation: true) do |response, index|
+    Parallel.each_with_index(responses.each, in_processes: num_procs,
```

Comment:
The change to `each` will mean all responses and answers are loaded into memory at once won't it?

Check this out: https://stackoverflow.com/a/57514911

Reply:
Theoretically yes, but in practice it changed nothing. I tried on my local machine with the full Health dataset and it ran smoothly either way. I wonder if Parallel batches internally.

Note: this change was necessary to support `order` above.